styles/components/avatar.css

```diff
@@ -0,0 +1,33 @@
+.avatar {
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    margin-right: 0.5rem;
```

Comment:
Try to use global variables for properties like margins, padding and border-radius in common cases

Reply:
ya i thought to do . thanks you . i ll do that

Reply:
Okay